packages/swapper/src/swappers/JupiterSwapper/swapperApi/getTradeQuote.ts

```diff
@@ -248,7 +248,7 @@ export const getTradeQuote = async (
         buyAmountAfterFeesCryptoBaseUnit: priceResponse.outAmount,
         sellAmountIncludingProtocolFeesCryptoBaseUnit: priceResponse.inAmount,
         jupiterQuoteResponse: priceResponse,
-        jupiterTransactionMetadata: {
+        solanaTransactionMetadata: {
```

Comment:
Renamed as it's not jupiter only but more a solana thing that we use for chainflip, relay and jupiter!

Reply:
See https://github.com/shapeshift/web/pull/9253/files#r2046458440 - seems to be Jupiter-specific? If it isn't, feel free to close, but that's the only place I could find references of `addressLookupTableAddresses`